docs/new_flux_ideas_and_guide.md

```diff
@@ -0,0 +1,985 @@
+also be wrapped in double quotes, which you'll need to use if they contain spaces:
+
+```
+@"my_data"."h2o_temperature"{"surface_degrees", "time"}
+```
+
+Given the previous data set, this query would produce a result like this: 
+
+| surface_degrees | time |
+| --- | --- |
+| 50.2 | 1600756160 |
+| 54.7 | 1600756160 |
+| 63.6 | 1600756160 |
+
+Since we didn't specify a time range in the query, we get the most recent data point for each series. How far 
+back in time the database looks for a recent value is dependent on the setup of the backend InfluxDB system you 
+are working with (this will typically be less than a few hours). We'll show how to specify explicitly how far 
+back to look for the most recent value in a moment.
+
+In this result set we have three rows, but we can't see their series information (i.e. the tags) because we 
+didn't select them. Selecting them is fairly easy:
+
+```
+@my_data.h2o_temperature{location, state, surface_degrees, time}
+```
+
+Would return the last value of surface_degrees in each series:
+
+| location | state | surface_degrees | time |
+| --- | --- | --- | --- |
+| coyote_creek | CA | 50.2 | 1600756160 |
+| puget_sound | WA | 54.7 | 1600756160 |
+| santa_monica | CA | 63.6 | 1600756160 |
+
+
+In Flux, a tag key and a field key are both called columns. This means that tag keys and field keys must
+be unique within a measurement. Here's an example that explicitly selects all the columns we know about:
+
+```
+@my_data.h2o_temperature{location, state, surface_degrees, bottom_degrees, time}
+``` 
+
+We can also select all columns by leaving the column list empty:
+
+```
+@my_data.h2o_temperature{}
+``` 
+
+Or you can leave out the curly braces:
+
+```
+@my_data.h2o_temperature
+```
+
+If your **Column Filter Predicate** selects only a tag, you'll get the same number of records as you have series for 
+that tag since the last value for each series is what gets defined if you're not filtering by time range or other 
+criteria.
+
+```
+@my_data.h2o_temperature{state}
+```
+
+Returns these results:
+
+| state |
+| --- |
+| CA |
+| CA |
+| WA |
+
+#### Filtering on Criteria
+You can limit the results returned in the query by specifying filtering criteria in the *Column Filter Predicate* block. 
+Here's an example that filters based on the location:
+
+```
+@my_data.h2o_temperature{location, surface_degrees, time, state == "CA"}
+```
+
+| location | state | surface_degrees | time |
+| --- | --- | --- | --- |
+| coyote_creek | CA | 50.2 | 1600756160 |
+| santa_monica | CA | 63.6 | 1600756160 |
+
+Note that `state` is returned in the result because it was listed in the filter criteria. You can have more complex 
+logic to match criteria like:
+
+```
+@my_data.h2o_temperature{location, surface_degrees, time, location == "coyote_creek" OR location == "puget_sound"}
+```
+
+See that while the column names are separated by commas, the filter criteria is separated by the different matching 
```

Comment:
I don't understand this sentence. Based on the example above, i would try `@my_data.h2o_temperature{location == "coyote_creek" OR location == "puget_sound", surface_degrees, time}` here.

Reply:
Me too.